.github/actions/conditional/action.yml

```diff
@@ -10,6 +10,9 @@ outputs:
   ci:
     description: Should "ci.yml" execute
     value: ${{ steps.changes.outputs.ci }}
+  ci-quarkus:
+    description: Should "ci.yml" execute
```

Comment:
Looks like this description was copy-pasted a couple of times, might warrant a more unique description, otherwise it's a tad redundant.

Reply:
Yeah, technically it's still concerning ci.yml but only some of its "subflows". Seemed a bit out of scope to change 
 it though, wanted to keep the PR focused. We can do as a follow-up. :)